lib/order.rb

```diff
@@ -0,0 +1,59 @@
+# I would like to be able to select some number of several available dishes
+# I would like to check that the total I have been given matches the sum of the various dishes in my order
+
+require_relative 'menu.rb'
+
+class Order
+
+attr_reader :basket, :balance
+
+  def initialize(menu= Menu.new)
+    # (menu_klass = Menu)
+    # @menu_klass = menu_klass
+    @basket = {}
+    @balance = 0
+  end
+
+  def place_order(item, quantity = 1)
+    fail "please order from menu" if !@menu.public_menu_contents.keys.include?(item)
+    basket << [item, quantity]
+  end
+
+  def price(item, amount)
+    total = 0
+    cost += (@public_menu_contents[item]*amount)
+  end
+
+#Here I want to check that the basket matches price.
+  def create_receipt
+  total_cost = 0
+  @basket.each {|item, quantity|
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)